samples/sensor/vl6180x/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _vl6180x:
+
+VL6180X: Time Of Flight sensor
```

Comment:
Please make this sample generic to any TOF sensor and add it in a separate commit

Reply:
@MaureenHelm 
Can you give me a little help - can you point me another generic sample?

For example:
So I has to change from vl6180x to tof? And define some other overloads?

I don't udnerstand :-(

Reply:
See `samples/sensor/accel_polling` and `samples/sensor/magn_polling`